Formula/q/qshell.rb

```diff
@@ -0,0 +1,27 @@
+class Qshell < Formula
+  desc "Shell Tools for Qiniu Cloud"
+  homepage "https://github.com/qiniu/qshell"
+  url "https://github.com/qiniu/qshell/archive/refs/tags/v2.14.0.tar.gz"
+  sha256 "1bb1eb9c7c71c0af7714fa49b04b5e27c851e49a0f8fb0360254cb3c9f4ee472"
+  license "MIT"
+  head "https://github.com/qiniu/qshell.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
```

Comment:
```suggestion

```

Reply:
This can be removed. Livecheck will check the git tags by default, so we only need this if we need to exclude certain tags.